docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+ASM1 Property Package
+=====================
+
+This package implements property relationships for the treatment of wastewater using an activated sludge biological reactor as provided in `Henze, M. et al. (1987) <https://belinra.inrae.fr/doc_num.php?explnum_id=4467>`_.
+
+This Activated Sludge Model no.1 (ASM1) property package:
+   * supports only 'H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', and 'S_ALK' as components
+   * supports only liquid phase
+   * is formulated on a mass basis
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', 'S_ALK']"
```

Comment:
I think after "Sets", it would be good to add another section for "Components" where we have a csv-table that gives the names for each component in connection with symbol names. For example:
"Active heterotrophic biomass",  ":math:`\text{X}_{b, h}`"

Reply:
Similar to what you have for Parameters, except we'd only need the Description and Symbol columns

Reply:
Nevermind- I see now that you already accounted for these in the "Properties" section!